include/soft_limiter.h

```diff
@@ -208,8 +186,8 @@ void SL_TEMPLATE_INST::Apply(in_array_t &in, out_array_t &out, const uint16_t re
 	//         static_cast<double>(peak.right));
 }
 
-SL_TEMPLATE_DEF
-void SL_TEMPLATE_INST::PrintStats(const AudioFrame<in_t> &external_scale) const
+template <size_t array_frames>
+void SoftLimiter<array_frames>::PrintStats(const AudioFrame<float> &external_scale) const
```

Comment:
Hmm, maybe there's no reason for `AudioFrame` to be a template class any more?

Reply:
Right; another nice  cleanup!